partcad/src/partcad/plugin_factory_provider_manufacturer.py

```
@@ -79,9 +79,14 @@ async def load(self, cart_item: ProviderCartItem):
         caps = await self.plugin.get_caps()
         # TODO(clairbee): Make the below more generic
         if "formats" in caps and "step" in caps["formats"]:
-            part: Part = self.ctx.get_part(cart_item.name)
+            # A cart item is a part most of the time, but it is an assembly
+            # whenever one is ordered assembled
+            object = resolve_cart_object(self.ctx, cart_item.name)
```

Ruff (0.16.1)

[error] Variable `object` is shadowing a Python builtin

(A001)